waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -131,13 +133,12 @@ npmDepsForWasp spec =
         AS.Dependency.fromList
           [ -- TODO: Allow users to choose whether they want to use TypeScript
             -- in their projects and install these dependencies accordingly.
-            ("typescript", "^4.8.4"),
-            ("@types/react", "^17.0.39"),
-            ("@types/react-dom", "^17.0.11"),
+            ("vite", "^4.1.0"),
+            ("typescript", "^4.9.3"),
+            ("@types/react", "^17.0.53"),
+            ("@types/react-dom", "^17.0.19"),
             ("@types/react-router-dom", "^5.3.3"),
-            -- TODO: What happens when react app changes its version? We should
-            -- investigate.
```

Comment:
This comment is still relevant, albeit for a different build system. Can we put it back?

Reply:
If we were to use a package with ome TS rules, yes. 

I'll investigate what'll we use, then I can add the comment :)

Reply:
Added this comment: https://github.com/wasp-lang/wasp/pull/1019/files#diff-5c849f0c343eac448a1ab375e436b53c29104a39f58d664f9334372f5ed6788aR145

Reply:
Thanks, even better :)